partcad-cli/src/partcad_cli/click/commands/reset.py

```
@@ -5,29 +5,50 @@
+    default=False,
+    help="Remove the runtime environments only.",
+)
+@click.option(
+    "--cache-only",
+    is_flag=True,
+    default=False,
+    help="Remove the filesystem caches only.",
+)
 @click.command(help="Reset all internal states maintained by PartCAD")
-def cli() -> None:
+def cli(repo_only: bool, runtime_only: bool, cache_only: bool) -> None:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)